Source/CarthageKit/Project.swift

```diff
@@ -354,6 +354,45 @@ public final class Project {
 			.collect()
 			.map(ResolvedCartfile.init)
 	}
+	
+	/// Attempts to determine which of the project's Carthage
+	/// dependencies are out of date.
+	///
+	/// This will fetch dependency repositories as necessary, but will not check
+	/// them out into the project's working directory.
+	public func outdatedDependencies(includeTransientDependencies: Bool) -> SignalProducer<[Dependency<OutdatedVersion>], CarthageError> {
```

Comment:
Also, we usually call these _nested_ dependencies. We should use that terminology here.